extensions/adobe/experience/customerJourneyManagement/message-delivery-feedback.schema.json

```diff
@@ -21,15 +21,60 @@
           "title": "Feedback Status",
           "type": "string",
           "description": "Provide status of a delivery attempt.",
-          "enum": ["error", "exclude", "send", "bounce", "delay"],
+          "enum": ["error", "exclude", "send", "bounce", "delay", "duplicate", "blocklisted"],
```

Comment:
Should this be blackListed?. Also we should also update the corresponding entry in meta:enum below.

Reply:
Blacklisted is not Adobe approved now.

It should be deny list as per latest adobe standards :)